.gitignore

```diff
@@ -18,5 +18,5 @@ coverage
 .vscode/*
 !.vscode/settings.json
 !.vscode/tasks.json
-!.vscode/launch.json
+.vscode/launch.json
```

Comment:
Why should i add my launch.json?
a file with exclamation mark means gitignore ignores it if i'm not mistaken

Reply:
if you are debugging inside vscode and would like to share your launch.json w/ others i think that's ok...i don't see the file included in your changeset though?